examples/curl.sh

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
+
+token=$(curl --request POST \
```

Comment:
May be worth putting in a comment saying what this curl command ends up doing (getting auth token), and doing the template render call before line 10. It's a bit verbose for sure, but may not be as self-evident for more junior developers reading the script their first time.